core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -1071,31 +1053,6 @@ public function makeRedirectPageMandatory(DataContainer $dc)
 		}
 	}
 
-	/**
-	 * Check for modified pages and update the XML files if necessary
-	 */
-	public function generateSitemap()
```

Comment:
Should we keep this method, deprecate it and redirect the call to `addSitemapCacheInvalidationTag()` for better backwards compatibilty?

Reply:
I don't think anyone should have ever called this method. It's a callback and not public API. The `Automator::generateSitemap()` is the API and that's kept for BC.

Reply:
I thought this callback might be useful to keep for BC because of a high chance that it was called from other callbacks (like `toggleVisibility` did).  
But I’m OK either way.